piwarsimulator.py

```diff
@@ -49,11 +51,24 @@
 running = True  # state of simulator
 ctrlc_count = 0  # if hitting 3 try and sys.exit
 
+class OperationMode(Enum):
```

Comment:
Yeah, this one's up to you. I didn't move MAP_LIST etc back because I'd already done the work in getting it into util before I realised it was moot. I can always pull it back into here.

The logic was that this should/could live here with MAP_LIST etc in the other one as the other two variables were shared across modules (this and the Vision360 module) whereas this Enum isn't, but since all the map and brain stuff is stringly typed anyhow, I ended up doing something different. I guess the real by-the-book solution would be to make it a proper Enum with appropriate helper functions, but we're under time pressure, so I'm leaving it as you had it before.

Reply:
OK I get your logic now. We can sort it out after merge if anything needs shifting back or something.